### DISCUSSION TITLE
"Workflows" section of "Actions" tab is not displaying correct workflow files

### DISCUSSION THREAD

**Discussion by mukherjeer-mms** [community/community#120092] [Actions] [answered]
### Select Topic Area

Question

### Body

As the title says, I am seeing different files in "Workflows" section of "Actions" tab than in the file tree of the repository, under .github/workflows folder of the default branch. Could someone please help me understand what's happening here?
![Screenshot 2024-04-18 at 16 04 27](https://github.com/community/community/assets/89006707/5ea6af48-64ca-4f4c-b2df-caa7ad0c2edd)
![Screenshot 2024-04-18 at 16 03 45](https://github.com/community/community/assets/89006707/f8529836-cb75-4d75-8a7a-4d97bb5ef788)

**Comment by SvenSchneiderDVAG** [comment, marked as answer]
The Action Tab shows the name given inside the workflow file. It's what defined under 
`name: My Test Workflow` on first line...

**Reply by grandmas-favorite** [reply]
Nice callout @SvenSchneiderDVAG! 😄
